README.md

```diff
@@ -33,36 +33,42 @@ clj-ldap is a thin layer on the [unboundid sdk](http://www.unboundid.com/product
 Connects to an ldap server and returns a, thread safe, [LDAPConnectionPool](http://www.unboundid.com/products/ldap-sdk/docs/javadoc/com/unboundid/ldap/sdk/LDAPConnectionPool.html).
 Options is a map with the following entries:
 
-    :host            Either a string in the form "address:port"
-                     OR a map containing the keys,
-                        :address   defaults to localhost
-                        :port      defaults to 389 (or 636 for ldaps),
-                     OR a collection containing multiple hosts used for load
-                     balancing and failover. This entry is optional.
-    :bind-dn         The DN to bind as, optional
-    :password        The password to bind with, optional
-    :num-connections The number of connections in the pool, defaults to 1
-    :ssl?            Boolean, connect over SSL (ldaps), defaults to false
-    :startTLS?       Boolean, use startTLS over non-SSL port, defaults to false
-    :trust-store     Only trust SSL certificates that are in this
-                     JKS format file, optional, defaults to trusting all
-                     certificates
-    :connect-timeout The timeout for making connections (milliseconds),
-    :timeout         The timeout when waiting for a response from the server
-                     (milliseconds), defaults to 5 minutes
+    :host             Either a string in the form "address:port"
+                      OR a map containing the keys,
+                         :address   defaults to localhost
+                         :port      defaults to 389 (or 636 for ldaps),
+                      OR a collection containing multiple hosts used for load
+                      balancing and failover. This entry is optional.
+    :bind-dn          The DN to bind as, optional
+    :password         The password to bind with, optional
+    :num-connections  Deprecated, use init-connections instead.
+    :init-connections The number of connections to initially establish when
```

Comment:
I would change it to `initial-connections` to match the api and the description:
"Establish a connection pool initially of this size with capability to grow to :max-connections. Defaults to 1."